paimon-core/src/main/java/org/apache/paimon/table/system/CatalogOptionsTable.java

```diff
@@ -93,6 +95,12 @@ public List<String> primaryKeys() {
         return Collections.singletonList("key");
     }
 
+    @Override
+    public FileIO fileIO() {
+        // pass a useless file io, should never use this.
+        return new LocalFileIO();
+    }
```

Comment:
Could throw UnsupportedOperationException here？

Reply:
I have considered this solution, but in compute engine side, maybe it will get FileIO and do something.